go.mod

```diff
@@ -8,158 +8,156 @@ require (
 	github.com/aws/amazon-vpc-resource-controller-k8s v1.0.7
```

Comment:
While we are updating, I think we need to update this to the latest VPC RC version

Reply:
Resolving this as it will be taken care of in issue #2229